app/tests/src/actions/wallet-actions.js

```diff
@@ -0,0 +1,93 @@
+import mockNetwork from '../common/mock'
+import {ADALITE_CONFIG} from '../../../frontend/config'
+import {CRYPTO_PROVIDER_TYPES} from '../../../frontend/wallet/constants'
+import mnemonicToWalletSecretDef from '../../../frontend/wallet/helpers/mnemonicToWalletSecretDef'
+import assert from 'assert'
+import {assertPropertiesEqual, setupState} from './actions'
+
+let state, action
+
+beforeEach(() => {
+  ;[state, action] = setupState()
+})
+
+const expectedStateChanges = {
+  walletIsLoaded: true,
+  balance: 1500000,
+  loading: false,
+  mnemonicAuthForm: {
+    mnemonicInputValue: '',
+    mnemonicInputError: null,
+    formIsValid: false,
+  },
+  usingHwWallet: false,
+  hwWalletName: undefined,
+  isDemoWallet: false,
+  showDemoWalletWarningDialog: false,
+  showGenerateMnemonicDialog: false,
+  // send form
+  sendAmount: {fieldValue: '', coins: 0},
+  sendAddress: {fieldValue: ''},
+  donationAmount: {fieldValue: '', coins: 0},
+  sendResponse: '',
+}
+
+it('Load shelley wallet', async () => {
```

Comment:
this test is timing on my machine, I would increase the timeout to a sfe value, let's say 5 seconds

Reply:
Increased just for this test. However it's a bit weird, when I run only this one test it runs 3x faster than when I uncomment the rest of the suite.

Reply:
Interesting, also not sure why, but when I run the tests locally and I include the test set `Import/Export Wallet as JSON`, I get the following error: 

```
(node:132999) UnhandledPromiseRejectionWarning: Error: Page crashed!
    at Page._onTargetCrashed (/home/rafael/workplace/cardano/adalite/app/node_modules/puppeteer/lib/Page.js:170:24)
    at CDPSession.Page.client.on.event (/home/rafael/workplace/cardano/adalite/app/node_modules/puppeteer/lib/Page.js:125:56)
    at CDPSession.emit (events.js:197:13)
    at CDPSession._onMessage (/home/rafael/workplace/cardano/adalite/app/node_modules/puppeteer/lib/Connection.js:200:12)
    at Connection._onMessage (/home/rafael/workplace/cardano/adalite/app/node_modules/puppeteer/lib/Connection.js:112:17)
    at WebSocketTransport._ws.addEventListener.event (/home/rafael/workplace/cardano/adalite/app/node_modules/puppeteer/lib/WebSocketTransport.js:44:24)
    at WebSocket.onMessage (/home/rafael/workplace/cardano/adalite/app/node_modules/ws/lib/event-target.js:120:16)
    at WebSocket.emit (events.js:197:13)
    at Receiver.receiverOnMessage (/home/rafael/workplace/cardano/adalite/app/node_modules/ws/lib/websocket.js:789:20)
    at Receiver.emit (events.js:197:13)
(node:132999) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:132999) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Are you able to replicate it?